packages/playground/public/info/jenkins.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Jenkins
+---
+
+Jenkins is a popular open-source automation server that enables developers to build, test, and deploy their applications continuously. For more details, check [Jenkins documentation](https://www.jenkins.io/doc/book/).
```

Comment:
The link should be the one produced from the manual. It will be ''https://www.manual.grid.tf/documentation/dashboard/solutions/jenkins.html"